src/app/ui-library/input.tsx

```diff
@@ -0,0 +1,236 @@
+import { ChangeEvent, useLayoutEffect, useRef } from 'react';
+import styled from 'styled-components';
+import { InputFeedback, type FeedbackVariant } from './inputFeedback';
+
+type InputVariant = 'danger' | 'default';
+
+const Container = styled.div`
+  display: flex;
+  flex-direction: column;
+`;
+
+const TitleContainer = styled.div`
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  margin-bottom: ${(props) => props.theme.space.xs};
+`;
+
+const Title = styled.div`
+  ${(props) => props.theme.typography.body_medium_m}
+  color: ${(props) => props.theme.colors.white_200};
+`;
+
+const InfoText = styled.div`
+  ${(props) => props.theme.typography.body_medium_m}
+  color: ${(props) => props.theme.colors.white_0};
+`;
+
+const InputContainer = styled.div`
+  position: relative;
+`;
+
+const InputField = styled.input<{ $variant: InputVariant }>`
+  ${(props) => props.theme.typography.body_medium_m}
+
+  width: 100%;
+
+  color: ${(props) => props.theme.colors.white_200};
+  background-color: ${(props) => props.theme.colors.elevation_n1};
+  padding: ${(props) => props.theme.spacing(5)}px ${(props) => props.theme.spacing(8)}px;
+  border-radius: 8px;
+
+  caret-color: ${(props) => props.theme.colors.tangerine};
+
+  ${(props) => {
+    switch (props.$variant) {
+      case 'danger':
+        return `
+          border: 1px solid ${props.theme.colors.danger_dark_200};
+        `;
+      default:
+        return `
+          border: 1px solid ${props.theme.colors.white_800};
+        `;
+    }
+  }}
+
+  &:focus {
+    ${(props) => {
+      switch (props.$variant) {
+        case 'danger':
+          return `
+            border: 1px solid ${props.theme.colors.danger_dark_200};
+          `;
+        default:
+          return `
+            border: 1px solid ${props.theme.colors.white_600};
+          `;
+      }
+    }}
+  }
+
+  ::placeholder {
+    color: ${(props) => props.theme.colors.white_400};
+  }
+`;
+
+const ComplicationsContainer = styled.div`
+  position: absolute;
+  right: ${(props) => props.theme.spacing(4)}px;
```

Comment:
```suggestion
  right: ${(props) => props.theme.spacing(8)}px;
```
<img width="534" alt="image" src="https://github.com/secretkeylabs/xverse-web-extension/assets/65149726/07d0fa57-97dd-4bbf-9964-6fd1c7bc0199">

Reply:
🙌